contrib/ohi-release-notes/action.yml

```diff
@@ -7,6 +7,9 @@ inputs:
   excluded-dirs:
     description: Exclude commits whose changes only impact files in specified dirs relative to repository root. Defaults to ".github".
     default: '.github'
+  excluded-files:
+    description: Exclude commits whose changes only impact files in specified files relative to repository root. Defaults to "".
+    default: ''
```

Comment:
Would it make sense to expose the included files and/or dirs in the `ohi-release-notes` action as well?

Reply:
I would keep the list as small as possible. Since we do not need them I would not add those for now.

I added them to the command just for "simmetry"

Reply:
Maybe I didn't understand it well, but I thought we already had a use case for `include-files`. Don't we want to take into account only changes in [infrastructure-bundle/bundle.yml](https://github.com/newrelic/infrastructure-bundle/blob/master/bundle.yml)?

Reply:
we said we were going to exclude only `go.sum` and `go.mod`, @kang-makes since you add a 👍 would you like as well for the infra-bundle to have just the include instead?

Reply:
I am reluctant to add not strictly needed options to the contrib action since there is the risk of having it growing and growing. Since it is like an action grouping all the other ones, we could easily end-up with "tens" of them :)

Reply:
I know that the contrib action might finish with lots of parameters and options but it should cover our use cases.

We have use-cases where we want to exclude the whole folder (like `.github` or `tools`) but we also have repositories where we want to exclude everything but a file (like the `bundle.yaml` of the infra bundle or `exporter.yaml` files in the exporter packages repository).

As we have many use cases, I would cover both: `exclude-directories` and `include-files` at least.

Reply:
By the way, since you are both interested into it and it is not that much toil I'll add them.


..... But I still think they will be unused 🤣